packages/test/test-version-utils/README.md

```diff
@@ -1,26 +1,137 @@
+are generated (empty entries are current versions):
+
+| Compat Kind         | Loader | Driver | Container Runtime | Data Runtime |
+| ------------------- | ------ | ------ | ----------------- | ------------ |
+| None                |        |        |                   |              |
+| Loader              |  old   |        |                   |              |
+| Driver              |        |  old   |                   |              |
+| ContainerRuntime    |        |        | old               |              |
+| DataRuntime         |        |        |                   | old          |
+| NewLoader           |        |  old   | old               | old          |
+| NewDriver           |  old   |        | old               | old          |
+| NewContainerRuntime |  old   |  old   |                   | old          |
+| NewDataRuntime      |  old   |  old   | old               |              |
+
+### Mocha test setup with layer version combinations
+
+There are three compat `describe*` to generate different combinations, depending of the need of the tests
+
+`describeFullCompat`: generate test variants with compat combinations that varies the version for all layers.
+
+- Used for tests that exercise all layers and will benefits compat combinations of all layers.
+
+`describeLoaderCompat`: generate test variants with compat combinations that only varies the loader version.
+
+- Use for tests that targets the loader layer, and don't care about compat combinations of other layers.
+- Test combination generated: [CompatKind.None, CompatKind.Loader]
+
+`describeNoCompat` - generate one test variant that doesn't varies version of any layers.
+
+- Use for tests that doesn't benefit or require any compat testing.
+- Test combination generated: [CompatKind.None]
+
+These compat `describe*` functions will also load the APIs with appropriate version and provide the test with a
+`TestObjectProvider` object, where the test can use to access Fluid functionality.  Even compat testing not
+necessary, `TestObjectProvider` provide convenience function and tracking capability, and it allows the test
+to enable compat testing easily in the future just by changing the `describe*`.
+
+### Legacy version defaults and installation
+
+By default, N-1, N-2, and LTS (hard coded) test variants are generated.  The versions can be specified using command
+line (see below) to run the test against any two versions. This package includes a `mocha` global hook that will
+install legacy packages at the beginning of the package based on the `compatVersion` settings.
+
+## Command line options
+
+Tests using the compat `describe*` will be controllable using the command line options when running mocha on
+driver selection, versions for compat testing, and compat kind combinations.
+
+```text
+--compatKind <CompatKind> - filter to the compat variant. See above table. Can specify multiple times.
```

Comment:
```suggestion
--compatKind <CompatKind> - filter to the compat variant. See above table. Can specify multiple kinds separated by comma.
```

Reply:
I think the flag can specify multiple time as well, not just one flag and compatkind in comma.

Reply:
Like `mocha --compatKind None --compatKind Loader`?  Maybe say `Can be specified multiple times` then